src/server/replica.cc

```diff
@@ -825,6 +844,12 @@ io::Result<bool> DflyShardReplica::StartSyncFlow(
                               CheckRespFirstTypes({RespExpr::STRING, RespExpr::STRING}));
 
   string_view flow_directive = ToSV(LastResponseArgs()[0].GetBuf());
+  // Fetch the LSN to consider partial sync complete
+  if (!(flow_directive.back() == 'L')) {
+    string_view lsn = flow_directive.substr(7);
```

Comment:
I went overboard with this to keep the metrics. IMO we can slightly change the definition of `partial_success` to fit logically with the idea that partial sync `is stable sync` just with a `few stale` journal changes. So, if we skipped `full_sync` it means partial sync is `successful` even though we are still `lagging behind because the journal changes did not reach us`.

Reply:
In fact if we follow this, we can remove a few lines :)